qiskit/wrapper/jupyter/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you make sure the new files' headers follows the structure of the rest of the files of the project? It's important that we are consistent, and order and newlines do matter:

```
# -*- coding: utf-8 -*-

# Copyright 201x, IBM.
#
# This source code is licensed under the Apache License, Version 2.0 found in
# the LICENSE.txt file in the root directory of this source tree.

# pylint: disable=missing-docstring,redefined-builtin

"""Module blah for foo and bar."""

import ...
```

Reply:
done